fetchmail_attach_from_folder/models/fetchmail_server_folder.py

```diff
@@ -33,73 +34,93 @@ class FetchmailServerFolder(models.Model):
+            "Examples:\n"
+            "- 'email' if your model is res.partner\n"
+            "- 'partner_id.email' if matching sale orders."
     )
     model_order = fields.Char(
-        "Order (model)",
-        help="Field(s) to order by, this mostly useful in conjunction "
-        "with 'Use 1st match'",
+        string="Model Order",
+        help="Specify the field(s) to order by when matching emails. "
+            "This is mostly useful in conjunction with 'Use 1st Match'."
     )
     match_algorithm = fields.Selection(
         selection=[
-            ("odoo_standard", "Odoo standard"),
-            ("email_domain", "Domain of email address"),
-            ("email_exact", "Exact mailadress"),
+            ("odoo_standard", "Odoo Standard"),
+            ("email_domain", "Domain of Email Address"),
+            ("email_exact", "Exact Email Address"),
         ],
         required=True,
-        help="The algorithm used to determine which object an email matches.",
+        string="Matching Algorithm",
+        help="Select the algorithm used to determine which object an email matches."
     )
     mail_field = fields.Char(
-        "Field (email)",
-        help="The field in the email used for matching."
-        " Typically this is 'to' or 'from'",
+        string="Email Field",
+        help="Specify the field in the email used for matching. "
+            "Typically, this is 'to', 'from' or 'reply_to'."
     )
     delete_matching = fields.Boolean(
-        "Delete matches", help="Delete matched emails from server"
+        string="Delete Matched Emails",
+        help="Enable this option to delete emails after they are successfully processed and matched."
     )
     flag_nonmatching = fields.Boolean(
+        string="Flag Non-Matching Emails",
         default=True,
-        help="Flag emails in the server that don't match any object in Odoo",
+        help="Enable this option to mark emails as important if they do not match any object in Odoo."
     )
     match_first = fields.Boolean(
-        "Use 1st match",
-        help="If there are multiple matches, use the first one. If "
-        "not checked, multiple matches count as no match at all",
+        string="Use First Match",
+        help="If there are multiple matches, use the first one. "
+            "If disabled, multiple matches will be considered as no match."
+    )
+    domain = fields.Char(
+        string="Matching Domain",
+        help="Define a search filter to narrow down objects for email matching."
     )
-    domain = fields.Char(help="Fill in a search filter to narrow down objects to match")
     msg_state = fields.Selection(
         selection=[("sent", "Sent"), ("received", "Received")],
         string="Message state",
         default="received",
         help="The state messages fetched from this folder should be assigned in Odoo",
     )
-    active = fields.Boolean(default=True)
+    active = fields.Boolean(
+        string="Active",
```

Comment:
Odoo will provide this capitalized name automatically, therefore should not be specified according to OCA guidelines.

Reply:
Thanks I did not know